README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
+
+Use [mermaid.js](https://mermaid-js.github.io/mermaid/#/) to create graphs and
+diagrams,
+[for example](https://www.ory.sh/docs/kratos/self-service/flows/user-registration).
+The [mermaid live editor](https://mermaid.live) runs in the browser.  
+To use the graphs in documents import Mermaid at the top of the document:
```

Comment:
I know we try to do this "imports on top" thing, but this hasn't been codified yet. As such, you're example is not consistent with other examples from the style guide that require imports. Please make it consistent and add a code block that has both the "import" line and the diagram.